channel/consensus_channel/consensus_channel.go

```diff
@@ -746,22 +758,23 @@ func (vars *Vars) Remove(p Remove) error {
 	vars.TurnNum += 1
 
 	// Adjust balances
-	o.left.amount.Add(o.left.amount, p.LeftAmount)
-	o.right.amount.Add(o.right.amount, p.RightAmount)
+	o.leader.amount.Add(o.leader.amount, p.LeftAmount)
+	o.follower.amount.Add(o.follower.amount, p.RightAmount)
 
 	// Remove the guarantee
 	delete(o.guarantees, p.Target)
 
 	return nil
 }
 
-// Remove is a proposal to remover a guarantee for the given virtual channel.
+// Remove is a proposal to remove a guarantee for the given virtual channel.
 type Remove struct {
+	// Target is the address of the virtual channel being defunded
 	Target types.Destination
-	// LeftAmount is the amount to be credited to the left participant of the two party ledger channel
+	// LeftAmount is the amount to be credited to the left funder of the guarantee for Target
 	LeftAmount *big.Int
-	// RightAmount is the amount to be credited to the right participant of the two party ledger channel
-	RightAmount *big.Int
+	// RightAmount is the amount to be credited to the right funder the guarantee for Target
+	RightAmount *big.Int // todo?: replace this with a function, as in Add
```

Comment:
Nit: I think `// TODO:` is preferable to `// todo?:`

Reply:
You are correct. This todo is on the immediate chopping block as well, so I'll leave it.